inputs/mixins.js

```diff
@@ -0,0 +1,37 @@
+export const DynamicEvents = {
+  directives: {
+    DynamicEvents: {
+      bind: function (el, binding, vnode) {
+        const allEvents = binding.value;
+
+        allEvents.forEach((event) => {
+          // register handler in the dynamic component
+          vnode.componentInstance.$on(event, (eventData) => {
+            // when the event is fired, the proxyEvent function is going to be called
+            vnode.context.proxyEvent(event, eventData);
+          });
+        });
+      },
+      unbind: function (el, binding, vnode) {
+        vnode.componentInstance.$off();
+      }
+    }
+  },
+  computed: {
+    customEvents() {
+      if (this.schema.events) {
+        return Object.keys(this.schema.events).map((key) => key );
+      }
+
+      return [];
```

Comment:
We could make this a ternary operator
```suggestion
      return this.schema.events
        ? Object.keys(this.schema.events).map((key) => key)
        : [];
```

Reply:
👍